pool/liquidity_math.gno

```diff
@@ -3,15 +3,11 @@ package pool
 import "gno.land/p/demo/ufmt"
 
 func liquidityMathAddDelta(x bigint, y bigint) bigint {
-	if y < 0 {
-		z := x - (-y)
-		requireUnsigned(z, ufmt.Sprintf("[POOL] liquidity_math.gno__liquidityMathAddDelta() #1 || expected z(%d) >= 0", z))
-		require(z < x, ufmt.Sprintf("[POOL] liquidity_math.gno__liquidityMathAddDelta() || expected z(%d) < x(%d)", z, x))
-		return z
-	} else {
-		z := x + y
-		requireUnsigned(z, ufmt.Sprintf("[POOL] liquidity_math.gno__liquidityMathAddDelta() #2 || expected z(%d) >= 0", z))
-		require(z >= x, ufmt.Sprintf("[POOL] liquidity_math.gno__liquidityMathAddDelta() || expected z(%d) >= x(%d)", z, x))
-		return z
-	}
+    z := x + y
```

Comment:
In the Solidity version of this code, x and y have type of uint and int respectively, which required the negation and applying the seemingly redundant operation. Commenting for the context.